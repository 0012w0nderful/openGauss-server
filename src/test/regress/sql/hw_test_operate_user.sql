create role test_myrole001 with password "gauss@123";
create role test_myrole002 with sysadmin password "gauss@123";

set role test_myrole001  password "gauss@123";
alter role test_myrole002 SET maintenance_work_mem = 100000; 
alter role test_myrole002 rename to temp_myrole;
reset role;
drop role test_myrole001;
drop role test_myrole002;
