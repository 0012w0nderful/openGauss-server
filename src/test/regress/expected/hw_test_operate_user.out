create role test_myrole001 with password "ttest@123";
create role test_myrole002 with sysadmin password "ttest@123";
set role test_myrole001  password "ttest@123";
alter role test_myrole002 SET maintenance_work_mem = 100000; 
ERROR:  Permission denied.
alter role test_myrole002 rename to temp_myrole;
ERROR:  Permission denied.
reset role;
drop role test_myrole001;
drop role test_myrole002;
